frontend/gpay_merchant/lib/business_location_page.dart

```diff
@@ -0,0 +1,256 @@
+import 'package:flutter/material.dart';
+import 'package:geocoder/geocoder.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'business_phone_page.dart';
+import 'location_confirm_on_map_page.dart';
+
+class BusinessDetailsPageTwo extends StatefulWidget {
+  @override
+  _BusinessDetailsPageTwoState createState() => _BusinessDetailsPageTwoState();
+}
+
+/*
+ * This class contains the view to accept the business address such as:
+ * It also redirects to a google maps page to confirm the location of the business.
+ */
+class _BusinessDetailsPageTwoState extends State<BusinessDetailsPageTwo> {
+  TextEditingController pincodeController = TextEditingController();
+  TextEditingController streetController = TextEditingController();
+  TextEditingController areaController = TextEditingController();
+  TextEditingController cityController = TextEditingController();
+  TextEditingController stateController = TextEditingController();
+  final formValidationKey = GlobalKey<FormState>();
+  bool validate = false;
+  bool isMapValid = true;
+
+  @override
+  Widget build(BuildContext context) {
+    double width = MediaQuery.of(context).size.width;
+    double height = MediaQuery.of(context).size.height;
+    TextStyle styleBold = GoogleFonts.montserrat(fontWeight: FontWeight.w500);
+    TextStyle style = GoogleFonts.montserrat();
+
+    showAlertDialog(String title, String alertMessage, BuildContext context) {
+
+      // set up the button
```

Comment:
comments should be a complete sentence with full stop.
https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences